metrics/multi_metrics_test.go

```diff
@@ -91,10 +91,10 @@ func TestMultiMetrics_Register(t *testing.T) {
 	mm.Down("updown")
 	mm.Gauge("gauge", 42)
 
-	// counter should be 0 because it's not tracked by StoreMetrics
+	// counter should be 1 because it's tracked by StoreMetrics
```

Comment:
@kentquirk , do you know why we didn't store counters in the `StoreMetrics`?

Reply:
They're not stored because they're reset on each metrics scrape, so they're not useful for stress relief.

https://github.com/honeycombio/refinery/blob/main/metrics/multi_metrics.go#L17

Reply:
Do you mind to point out where this values map resets? I don't think I understand how the reset works for them

Reply:
The values map doesn't reset, but counters do. The value you read from the values map won't reflect the value sent to Honeycomb because that one resets but the local values map doesn't.

Reply:
Ok, that aligns with my understanding.I will update the comments so it's clear that this is also used by the opamp agent now